src/hn/ui/CommentItem.ui

```diff
@@ -1,20 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.22.2 -->
 <interface>
-  <requires lib="gtk+" version="3.20"/>
+  <requires lib="gtk" version="4.0"/>
   <template class="CommentItem" parent="GtkBox">
-    <property name="visible">True</property>
-    <property name="can_focus">False</property>
+    <property name="can_focus">0</property>
     <property name="orientation">vertical</property>
     <child>
       <object class="GtkBox" id="comment_body">
+        
         <property name="orientation">vertical</property>
-        <property name="visible">True</property>
-        <property name="can_focus">False</property>
+        <property name="can_focus">0</property>
         <child>
           <object class="GtkLabel" id="time_lbl">
-            <property name="visible">True</property>
-            <property name="can_focus">False</property>
+            
+            <property name="can_focus">0</property>
```

Comment:
You probably want `focusable` here if you really don't want users to focus on these labels.